.ci/atime/tests.R

```diff
@@ -132,8 +132,9 @@ test.list <- atime::atime_test_list(
          val = rnorm(N))
      },
      expr = data.table:::`[.data.table`(dt, , .(vs = (sum(val))), by = .(id)),
-     Before = "be2f72e6f5c90622fe72e1c315ca05769a9dc854", # Parent of the regression causing commit (https://github.com/Rdatatable/data.table/commit/e793f53466d99f86e70fc2611b708ae8c601a451) in the PR that introduced the issue (https://github.com/Rdatatable/data.table/pull/4491/commits)
-     Regression = "e793f53466d99f86e70fc2611b708ae8c601a451", # Commit responsible for regression in the PR that introduced the issue (https://github.com/Rdatatable/data.table/pull/4491/commits)
-     Fixed = "58409197426ced4714af842650b0cc3b9e2cb842") # Last commit in the PR that fixed the regression (https://github.com/Rdatatable/data.table/pull/5463/commits)
+     Before = "fda7fd9dbcac939df33769193eebd44731ee59b9", # Parent of the regression commit (https://github.com/Rdatatable/data.table/commit/c4a2085e35689a108d67dacb2f8261e4964d7e12)  
```

Comment:
parent of this Regression commit (c4a2085e35689a108d67dacb2f8261e4964d7e12) is not appropriate here, because this Regression commit is the parent of the first commit in the fix PR (not the commit which caused the regression), so the parent of this Regression commit could still be slow. 

If the previous Regression commit (e793f53466d99f86e70fc2611b708ae8c601a451) is the commit which caused the regression, then it would be appropriate to use its parent as Before.

Please revert this commit, and write proper documentation comments for the commit IDs you were using before.